warehouse/models/staging/gtfs/_src_gtfs_rt_external_tables.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 sources:
   - name: external_gtfs_rt
     description: Hive-partitioned external tables reading GTFS RT data and validation errors from GCS.
-    database: "{{ env_var('GOOGLE_CLOUD_PROJECT', var('GOOGLE_CLOUD_PROJECT')) }}"
+    database: "{{ env_var('GOOGLE_CLOUD_PROJECT', var('EXTERNAL_TABLE_SOURCE')) }}"
```

Comment:
PR description says that GTFS RT data is meant to be excluded from this change, won't this update mean that it applies to GTFS-RT too? Might remove this change so that GTFS-RT is excluded, or update PR description to be clear that it affects all sources